MetaMask/metamask-extension
# Set account nickname to ENS when is available

Fixes: #11273 

Explanation:  If the user has not changed the account nickname and ENS is available, ENS reverse-resolved name is displayed on Account detail view instead of the nickname such as 'Account 1'. 

Manual testing steps:  
  - Import the account registered in ENS to Metamask
  - Set the alias of the account to a default value such as 'Account 1'
  - Go to  the Account detail view and check if the ENS value is loaded.
